src/docs/truffle/getting-started/installation.md

```diff
@@ -18,3 +18,133 @@ Truffle also requires that you have a running Ethereum client which supports the
 ## Recommendations for Windows
 
 If you're running Truffle on Windows, you may encounter some naming conflicts that could prevent Truffle from executing properly. Please see [the section on resolving naming conflicts](/docs/advanced/configuration#resolving-naming-conflicts-on-windows) for solutions.
+
+## Recommendations for Linux
+
+I you're running Truffle on Linux or MacOS X, you should not install it with sudo, otherwise you might encounter some permission issues. It is recommended to install truffle as a normal user. Before we can install truffle as a normal user we have to cleanup the system. It is also recommended to install nodejs and npm with nvm (node version manager). An advantage of nvm is that you can run different node versions on the same machine.
```

Comment:
```suggestion
If you're running Truffle on Linux or macOS, you should not install it with `sudo`, otherwise you might encounter some permission issues. It is recommended to install Truffle as a normal user. Before we can install Truffle as a normal user, we have to clean-up the system. It is also recommended to install Node.js and NPM with [NVM (Node Version Manager)](https://github.com/creationix/nvm]. An advantage of NVM is that you can run different Node versions on the same machine.
```